content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "माइक्रोप्रोफाइल, कॉन्फिगमैप्स और सीक्रेट्स का उपयोग करके कॉन्फिगरेशन को लागू करना"
+content_type: ट्यूटोरियल
+weight: 10
+---
+
+<!-- overview -->
+
+इस ट्यूटोरियल में आप सीखेंगे कि अपने माइक्रोसर्विस के कॉन्फ़िगरेशन को कैसे और क्यों लागू करना है। 
+विशेष रूप से, आप सीखेंगे कि एनवायरमेंट वेरिएबल सेट करने के लिए कुबेरनेट्स कॉन्फिगमैप्स और सीक्रेट्स का 
+उपयोग कैसे करें और फिर माइक्रोप्रोफाइल कॉन्फिग का उपयोग करके उनका उपभोग करें। 
+
+
+## {{% heading "prerequisites" %}}
+
+### कुबेरनेट्स कॉन्फ़िगमैप्स और सीक्रेट बनाना
+
+कुबेरनेट्स में डॉकर कंटेनर के लिए एनवायरमेंट वेरिएबल सेट करने के कई तरीके हैं, जिनमें शामिल हैं: डॉकरफाइल, 
+कुबेरनेट्स.वाईएमएल, कुबेरनेट्स कॉन्फिगमैप्स और कुबेरनेट्स सीक्रेट्स। ट्यूटोरियल में, आप सीखेंगे कि अपने 
+एनवायरमेंट वेरिएबल सेट करने के लिए कुबेरनेट्स कॉन्फिगमैप्स और कुबेरनेट्स सीक्रेट्स का उपयोग कैसे करें, जिनके वैल्यू
+आपके माइक्रोसर्विसेज में इंजेक्ट किए जाएंगे। कॉन्फिगमैप्स और सीक्रेट्स का उपयोग करने का एक लाभ यह है कि 
+उन्हें कई कंटेनरों में फिर से उपयोग किया जा सकता है, जिसमें विभिन्न कंटेनरों के लिए अलग-अलग एनवायरमेंट वेरिएबल
+को सौंपा जाना भी शामिल है।
+
+कॉन्फिगमैप्स एपीआई ऑब्जेक्ट हैं जो गैर-गोपनीय key-value पेयर्स को संग्रहीत करते हैं। 
+इंटरएक्टिव ट्यूटोरियल में आप सीखेंगे कि एप्लिकेशन के नाम को संग्रहीत करने के लिए 
+कॉन्फिग मैप का उपयोग कैसे करें। कॉन्फ़िगमैप्स के संबंध में अधिक जानकारी के लिए, 
+आप [दस्तावेज़ यहाँ पा सकते हैं](/docs/tasks/configure-pod-container/configure-pod-configmap/))।
+
+हालाँकि सीक्रेट्स का उपयोग key-value पेयर्स को संग्रहीत करने के लिए भी किया जाता है, 
+वे कॉन्फिगमैप्स से भिन्न होते हैं क्योंकि वे गोपनीय/संवेदनशील जानकारी के लिए होते हैं और Base64 एन्कोडिंग
+का उपयोग करके संग्रहीत होते हैं। यह सीक्रेट क्रेडेंशियल्स, keys और टोकन जैसी चीज़ों को संग्रहीत करने
+के लिए उपयुक्त विकल्प बनाता है, जिनमें से पहला काम आप इंटरैक्टिव ट्यूटोरियल में करेंगे। सीक्रेट के बारे 
+में अधिक जानकारी के लिए, आप [दस्तावेज़ यहाँ पा सकते हैं](/docs/concepts/configuration/secret/)।
+
+### कोड से कॉन्फ़िग को लागू करना
+बाहरी एप्लिकेशन कॉन्फ़िगरेशन उपयोगी है क्योंकि कॉन्फ़िगरेशन आमतौर पर आपके वातावरण के आधार पर 
+बदलता है। इसे पूरा करने के लिए, हम जावा के संदर्भ और निर्भरता इंजेक्शन (सीडीआई) और माइक्रोप्रोफाइल 
+कॉन्फ़िगरेशन का उपयोग करेंगे। माइक्रोप्रोफाइल कॉन्फिग माइक्रोप्रोफाइल की एक विशेषता है, जो क्लाउड-नेटिव 
+माइक्रोसर्विसेज को विकसित करने और तैनात करने के लिए खुली जावा प्रौद्योगिकियों का एक सेट है।
+
+सीडीआई एक मानक निर्भरता इंजेक्शन क्षमता प्रदान करता है जो किसी एप्लिकेशन को सहयोगात्मक, शिथिल-युग्मित 
```

Comment:
Are we sure "शिथिल-युग्मित" is a term any common person knowing Hindi would be able to understand?

Reply:
How about: " CDI एक एप्लिकेशन को अलग-अलग हिस्सों (जिन्हें बीन्स कहते हैं) को मिलाकर बनाने की अनुमति देता है जो आपस में कड़ी तरह से जुड़े नहीं होते। यह काम एक मानक प्रक्रिया के जरिए होता है जिसे डिपेंडेंसी इंजेक्शन कहते हैं। " ?

Reply:
सीडीआई (CDI) एक स्टैंडर्ड तरीका है जो एप्लिकेशन में डिपेंडेंसी इंजेक्शन (dependency injection) को आसान बनाता है। इसकी मदद से, एप्लिकेशन को अलग-अलग हिस्सों (beans) से मिलाकर बनाया जा सकता है जो एक-दूसरे से कम जुड़े होते हैं। इससे एप्लिकेशन को बनाना और सुधारना आसान हो जाता है।

Think this sentence is easy to understand. Think on it

Reply:
@niranjandarshann Agreed. I'll replace these lines.